src/modules/PublicLab.MainImageModule.js

```diff
@@ -132,6 +133,11 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
       progressall: function (e, data) {
 
         var progress = parseInt(data.loaded / data.total * 100, 10);
+        dragImageI.innerHTML = "";
+        
+        // For hiding the HTML "Drag an image here to upload." after uploading image.
+        dragImageI.innerHTML = "";       
```

Comment:
Oh hm, is this line duplicated form 136? And, would you mind uploading a GIF of this working just so we can be sure? Thank you!